api/mesh/options.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+
+package kuma.mesh;
+
+option go_package = "github.com/kumahq/kuma/api/mesh";
+
+import "google/protobuf/descriptor.proto";
+
+message KumaResource {
```

Comment:
we already have a message of such name in `kds.proto`. I know it's technically a different package but how about `KumaResourceDescriptor` instead?

Reply:
Renamed to `KumaResourceOptions`.